src/OrchardCore.Modules/OrchardCore.Facebook/Views/FacebookSettings.Edit.cshtml

```diff
@@ -6,6 +6,11 @@
     <a class="seedoc" href="@(OrchardCore.Admin.Constants.DocsUrl)reference/modules/Facebook/#core-components" target="_blank">@T["See documentation"]</a>
 </p>
 
+@if (Model.CouldNotDecryptSettings)
+{
+    <p class="alert alert-danger">@T["An error occurred while decrypting the Secret Key, so it has been left blank. Anything using these settings will not currently work."]</p>
```

Comment:
```suggestion
    <p class="alert alert-danger">@T["An error occurred while decrypting a setting, so the feature has been disabled. Please enter and save to enable this feature."]</p>
```

Reply:
Should it say that the feature is disabled, since that's an actual function of Orchard, or should it maybe say "...the feature is non-functional" or something similar instead?

Reply:
Well generally the `Options` catches the exceptions, and just stops working, so is is close enough to a feature being disabled, i.e. it doesn't work. 
But you're right it's not technically a feature disabled. Non-functional is a bit too techy (for me) for the ui. But if you can come up with a better way of saying it please do.

Or just keep it shorter and don't say so much (it was just really long, and a lot of information)
`An error occurred while decrypting a setting. Please apply and save.`

Normally something is logged by the options class, so for more info users should be looking in logs - not that anyone ever remembers to check there ;)

Reply:
I updated the text to the shorter message.